cmd/kubernetes.go

```diff
@@ -192,17 +192,17 @@ var execCredentialCmd = &cobra.Command{
 				Status: &clientauth.ExecCredentialStatus{
 					ClientKeyData:         string(clientKeyData),
 					ClientCertificateData: string(clientCertificateData),
-					ExpirationTimestamp:   &metav1.Time{Time: time.Now().Add(time.Hour)},
+					ExpirationTimestamp:   &metav1.Time{Time: *expirationTime},
```

Comment:
I think you should check the case "if `expirationTime` is empty (or nil), you'd better set `time.Now().Add(time.Hour)`".

Reply:
Okay, will add this.

Reply:
Done in fd88b5e